solidity/contracts/liquidity-protection/LiquidityProtectionSettingsMigrator.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "../utility/Owned.sol";
+
+interface ISettings {
+    function addPoolToWhitelist(address pool) external;
+    function setNetworkTokenMintingLimit(address pool, uint256 limit) external;
+    function renounceRole(bytes32 role, address account) external;
+}
+
+contract LiquidityProtectionSettingsMigrator is Owned {
+    function migrate(ISettings settings, address[] calldata pools, uint256[] calldata limits) external ownerOnly {
```

Comment:
not clear why it requires the list of pools as an arg, can't just use the whitelist and read the values from the current (old) settings contract?

Reply:
Explained.